lmdeploy/serve/async_engine.py

```diff
@@ -470,10 +472,16 @@ async def generate(
         prompt = messages
         if do_preprocess:
             prompt = self.chat_template.messages2prompt(prompt, sequence_start)
+        logger.warning(f'Prompt with applied chat template:\n{prompt}')
```

Comment:
May use info-level log

Reply:
The info-level log will print a prompt for every call of the function. Besides, the default log-level is `warning` now.